docs/tutorialkit.dev/src/content/docs/reference/theming.mdx

```diff
@@ -48,12 +48,22 @@ These tokens can be used to style the colors of the tutorial on a high level. Fo
 | `--tk-elements-app-textColor`           | The text color of the TutorialKit app.       |
 | `--tk-elements-app-backgroundColor`     | The background color of the TutorialKit app. |
 | `--tk-elements-app-borderColor`         | The border color of the TutorialKit app.     |
-| `--tk-elements-app-linkColor`           | The link color of the TutorialKit app.       |
 
 If you want to start with a `theme.css` file that includes all the available CSS variables you can use the following snippet:
 
 <Code code={themeCSS} lang="css" title="theme.css" class="limit-code-height" />
 
+### Links
+
+TODO What here?
```

Comment:
These links will affect both `app` and `content`. There are tokens for `app` and `content`, but these links will be separate one. I'm not sure if this is the right approach.

Reply:
I think every approach has its merit but I prefer the one we're taking here.

It's all a hiearchy anyway and we could have kept `App > Links` and `Content > Links` but I don't think it makes much sense in practice as modern design tends to recommend using consistent colors for links across your application to maximize accessibility and UX (easier to guess it's a link). I learned that reading [Practical UI](https://www.practical-ui.com/) (an excellent read that I highly recommend!).

Here we're changing the logic, to just `Links` tokens. However we could have later, _if we really need it_, `Links > App` and `Links > Content` to be able to tweak those further. So global configuration then more specific.